eva/catalog/catalog_utils.py

```diff
@@ -91,12 +92,29 @@ def get_image_table_column_definitions() -> List[ColumnDefinition]:
     ]
     return columns
 
+def get_pdf_table_column_definitions() -> List[ColumnDefinition]:
+    """
+    id: pdf id
+    data: pdf data
+    """
+    columns = [
+        ColumnDefinition(PDFColumnName.id.name, ColumnType.INTEGER, None, None),
+        ColumnDefinition(
+            PDFColumnName.data.name,
+            ColumnType.NDARRAY,
+            NdArrayType.UINT8,
+            (None, None, None),
```

Comment:
Why the pdf data column is NDARRAY of UINT8?

Reply:
As the data will be the text content and the array will contains the paragraphs of the page in the array

Reply:
Would that be `NdArrayType.STR`? On the other hand, pdf != text. Is the plan here to have an internal reader that only extracts the text from the pdf? A few more questions, it is completely fine if we haven't thought about it. Why we partition the text based on the paragraphs instead of sentences, words, or even the whole pdf as single string? Based the query, the needs can be different. And, do we have examples queries on how we plan to query pdf in EVA?

Reply:
yes it should be NdArrayType.STR, yes I had a discussion with professor joy where we will be  having  a internal reader that only extracts the text from the pdf.
yes , we can have the text based on sentences and words and also a single string based on pages.
we plan to use the EVA query with load pdf where it create a table which have the text items as per the requirements from the pdf